### DISCUSSION TITLE
Check interval is not regular

### DISCUSSION THREAD

**Discussion by wolfier74** [upptime/upptime#278] [Q&A] [answered]
Hello, why check interval is not regular every 5 mins? Sometimes it just checks 1 or 2 times per hour.

Is there a way how to make it regular?

**Comment by AnandChowdhary** [comment, marked as answer]
Please see https://github.com/upptime/upptime/issues/224, https://github.com/upptime/upptime/issues/42#issuecomment-765174951, or my post at https://upptime.js.org/blog/2021/01/22/github-actions-schedule-not-working/.
